### ISSUE TITLE
Random Crash

### ISSUE THREAD

**Issue by verticaljelly** [anegostudios/VintageStory-Issues#207]
I think it's completely random but here is the log

```
Running on 64 bit Windows with 32GB RAM 
Version: 1.7.3 Survival Mode Edition (Stable)08/10/2018 14:13:14: Critical error occurred
System.Exception: The dialogue GuiDialogBlockEntityRecipeSelector was not added yet. Missing call to api.Gui.RegisterDialog()
   at _oe0xDD7HcFrqf4Uc1GxdYnZw4zg._EcW5KTJ2nW4hVprg5PuBfppUUJn(GuiDialog )
   at _oe0xDD7HcFrqf4Uc1GxdYnZw4zg._0RxpNklXMDQiTeMOtWHqiieNhyh(MouseEvent )
   at _foI2hCxGcDzKqqjfeABpB6CpNBLA._0RxpNklXMDQiTeMOtWHqiieNhyh(MouseEvent )
   at _UHOboDhqFr8kkxnQcdyFGkcIt2T._YCKd3wOG0lXFKz4lalS3nUCWmk5(Object , MouseButtonEventArgs )
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   at OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   at OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _veX1k27ELMT3YEQeuy3VRTN4yRb._kzSkvioOdlPvAytfREVDL8TchXc(_O2vbPFDX8XrsU0KZbWpNtJoa8sy , String[] )
   at _ggZEaGVfcLlJoMgWF2gpbmwaE3z._kzSkvioOdlPvAytfREVDL8TchXc(ThreadStart )
```

**Comment by tyronx** [comment]
Aw, this one again. Could you roughly explain what you did just before the crash?

**Comment by tyronx** [comment]
Is this bug still happening? I think I fixed this in one of the later v1.7.x versions.

**Comment by tyronx** [comment]
No feedback supplied in 2+ months. Assuming fixed.
